### ISSUE TITLE
메모리 제한에 대한 질문

### ISSUE THREAD

**Issue by Jaeminiman** [LISTatSNU/FastMRI_challenge#195]
안녕하세요, 
메모리 제한이 8GB인 점에 대해 궁금하여 질문합니다.

메모리 제한이 학습할 때 차지하는 GPU memory가 8GB를 넘지 않아야 하는 것인지,
학습된 모델을 가져와서 Reconstruction을 진행하는 과정에서 8GB를 넘지 않아야 하는 것인지 기준을 모르겠어서 질문드립니다.

감사합니다.

**Comment by wogur110** [comment]
안녕하세요, 2023 SNU FastMRI Challenge 조교 배재혁입니다.
GPU 8GB 메모리 제한은 train - evaluation 과정 모두 적용됩니다. 즉, 여러분이 제공 받은 Vessl GPU Server (8GB GPU) 에서 모델을 학습시키고, 리더보드 데이터셋을 reconstruction 진행할 수 있어야 합니다.

감사합니다.
배재혁 드림

**Comment by csymhph** [comment]
안녕하세요, workspace metadata를 보면 할당된 GPU memory가 14GiB로 나와 있고, resource monitor를 보면 학습 중에 12~13GiB까지도 올라간다고 나오는데, 실제 학습에 가용한 GPU memory는 8GiB이고 실제로도 그 한도 내에서 돌아가고 있지만, 학습 과정 외에 시스템상 디폴트로 사용되는 리소스로 인해 더 많이 사용되는 것처럼 표시되는 것이라고 생각하면 될까요?

**Comment by wogur110** [comment]
네 맞습니다. 실제 Vessl GPU server에서 사용하고 있는 GPU는 GTX1080으로, 8GB의 GPU memory를 가지고 있습니다. 해당 GPU에서 학습 및 reconstruction이 모두 이루어지면 됩니다.
